lib/commands/databases.js

```diff
@@ -2364,11 +2368,7 @@ const databasesGetDatabaseUsage = async ({databaseId,range,parseOutput = true, o
     }, payload);
 
     if (parseOutput) {
-        if(console) {
-            showConsoleLink('databases', 'getDatabaseUsage', databaseId);
```

Comment:
I think this was feature we added recently, for some of methods that show very ugly in CLI, so there is param to give you Console URL instead, to view it in the UI.

Any reason why we are removing this? If so, wasn't there more places we need to remove it from?